src/senaite/core/upgrade/v02_06_000.py

```diff
@@ -1641,6 +1641,42 @@ def migrate_attachmenttypes_to_dx(tool):
     logger.info("Convert AttachmentTypes to Dexterity [DONE]")
 
 
+@upgradestep(product, version)
+def move_dynamicanalysisspecs(tool):
+    """Move dynamic analysis specs to senaite package
+    """
+
+    # run required import steps
+    tool.runImportStepFromProfile(profile, "typeinfo")
+    tool.runImportStepFromProfile(profile, "workflow")
+
+    # get the old container
+    origin = api.get_setup().get("dynamic_analysisspecs")
+    if not origin:
+        # old container is already gone
+        return
+
+    # get the destination container
+    destination = get_setup_folder("dynamic_analysisspecs")
```

Comment:
Can you change the folder name to `dynamicanalysisspecs` to match the naming convention of the other DX setup folders?

Reply:
yes, indeed, noticed about convention